traits/observation/_has_traits_helpers.py

```diff
@@ -170,3 +113,30 @@ def observer_change_handler(event, graph, handler, target, dispatcher):
             dispatcher=dispatcher,
             remove=False,
         )
+
+
+def ctrait_prevent_event(event):
+    """ Return true if the CTrait change event should be skipped.
+
+    Parameters
+    ----------
+    event : TraitChangeEvent
+
+    Returns
+    -------
+    skipped: bool
```

Comment:
reST / numpydoc nit:
```suggestion
    skipped : bool
```